.editorconfig

```diff
@@ -1,3 +1,8 @@
+// SPDX-FileCopyrightText: 2020, Torsten Grote <t@grobox.de>
```

Comment:
Is there a way to maybe not put my name and especially the email address in each file?

I usually avoid putting my email address in clear text like this in the internet to avoid SPAM bots from crawling it and bombarding me with even more SPAM.

Reply:
Yes, if the project provides already contact information for legal inquiries I don't see much trouble in hiding the email on source files. However, with copyright is more complex, because we need a clear information on whom the code belongs. If you prefer, you could use a pseudonym or a nickname that could relate to you or the copyright holder.

Reply:
That would be nice! @lnceballosz could you please make that change? The full name is fine, just the email address could be removed.

Reply:
Sure, I will submit a new PR with those changes when time allows.

Reply:
Great, thanks! Feel free to change things within this MR though. You can just add more commits and we squash them together when merging.